draft-ietf-httpbis-digest-headers.md

```diff
@@ -263,20 +264,19 @@ It can be used in both request and response.
 The resource is specified by the effective request URI and any `validator`
 contained in the message.
 
-The relationship between Content-Location (see [RFC7231] Section 3.1.4.2)
-and Digest is demonstrated in {{post-not-request-uri}}.
-A comprehensive set of examples showing the impacts of
-representation metadata, payload transformations and HTTP methods on digest
-is provided in {{examples-unsolicited}} and {{examples-solicited}}.
+The relationship between Content-Location (see Section 6.2.5 of
+[I-D.ietf-httpbis-semantics]) and Digest is demonstrated in
```

Comment:
Like above, the CRLF at line 268 doesn't break the line in HTML nor in the TXT
so I won't reflow that.

Reply:
This is one paragraph. Having arbitrary line breaks in a contiguous block of text is counter intuitive, why would we want that?.

Text reflow in my editor applies to paragraphs, so if you want to maintain arbitrary line breaks that makes my life way harder and I just don't see the benefit.

Reply:
I generally try to isolate document lines that may be changed together - eg 'It can be used in request and response' - because I thought it simplifies editing.

If it doesn't and instead you find that makes things harder it's fine to let the editor reflow :)